policies/travel-policy.md

```diff
@@ -29,9 +29,7 @@ Definitions
 -   **Travel** is where; you leave your usual state and/or country or you
     have to travel 1 or more hours away from the location you usually
     work from.
--   **Travel Approver**: is a person authorised by the OMC to approve
-    travel expenses up to the limits as stated in the *Travel Approver
-    Table*.
+-   **Travel Approver**: is a person (contractor's manager) authorised by the OMC or OMC itself to approve travel expenses up to the limits as stated in the *Travel Approver Table*.
```

Comment:
Considering that the language further down essentially says, "Travel Approver or the OMC", this change seems unnecessary, and potentially confusing.

Reply:
Adjusted the language further down so it refers to "Travel Approver" without the additional explanation & left the definition as it.